### DISCUSSION TITLE
Mutect2 joint somatic SNV calling and SRC

### DISCUSSION THREAD

**Discussion by tnyamaguchi** [uclahs-cds/pipeline-call-sSNV#110] [General]
https://gatk.broadinstitute.org/hc/en-us/community/posts/360077166571-Mutect2-Joint-calling

I found this post and for SRC, Mutect2 somatic joint calling may not work well depending on the tree structure. (maybe recommending individual N/T calls or tweak options related to sensitivity?..) @yashpatel6 @lydiayliu @maotian06 @whelena @dan-knight 

```
Multi-sample mode is usually more sensitive because it is able to call variants that have mediocre evidence in several samples.
However, it can be less sensitive in the case where only one sample has a variant and the evidence in that sample is barely sufficient.
In such a case the preponderance of non-variant reads in the other samples dilutes the signal.

To some extent this is mathematically appropriate, because seeing several samples without a variant should make you less confident.
However, our multisample genotyping is definitely not as sophisticated as it could be and the dilution effect I mentioned above would be weaker with a more rigorous model.
```

**Comment by lydiayliu** [comment]
Interesting! Though one could argue that the `the case where only one sample has a variant and the evidence in that sample is barely sufficient` is also the least interesting case for subclonal reconstruction, considering that you would expect every sample to harbour some unique variants, and those unique variants just end up as leaves anyway and don't affect tree building.

One could always run a few benchmarking to see the call set difference from join vs individual N/T calls... As usual I recommend taking the union of individual N/T calls and running allele counting, though that call set is probably mathematically less rigorous and lets in false positives from the allele counting.
